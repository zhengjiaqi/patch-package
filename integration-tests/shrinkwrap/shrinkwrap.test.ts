import { runIntegrationTest } from "../runIntegrationTest"
runIntegrationTest({ projectName: "shrinkwrap", shouldProduceSnapshots: true })
