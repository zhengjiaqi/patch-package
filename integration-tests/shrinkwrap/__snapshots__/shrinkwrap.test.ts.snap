// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test shrinkwrap: left pad should contain patch-package 1`] = `
"SNAPSHOT: left pad should contain patch-package
module.exports = patch-package;
END SNAPSHOT"
`;
